bodhi/server/models.py

```diff
@@ -735,6 +735,75 @@ def unpush(self, koji):
 
 
 class Update(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        title (unicode): The update's title which uniquely identifies the update.
+        autokarma (bool): A boolean that indicates whether or not the update will
+            be automatically pushed when the stable_karma threshold is reached.
+        stable_karma (int): A positive integer that indicates the amount of "good"
+            karma the update must receive before being marked as stable.
```

Comment:
s/marked/automatically marked/ (The update can still be manually marked as stable after 7 days.)